test/admin-hpa/features/support/received_check_decompressor.rb

```diff
@@ -3,13 +3,22 @@
 
 class ReceivedCheckDecompressor
 
-  credentials = File.read('../../admin/.env').split('AZURE_STORAGE_CONNECTION_STRING').last.split(';')
-  @account_name = credentials.find{|a| a.include? 'AccountName' }.gsub('AccountName=','')
-  @account_key = credentials.find{|a| a.include? 'AccountKey' }.gsub('AccountKey=','')
-  AZURE_TABLE_CLIENT = Azure::Storage::Table::TableService.create(storage_account_name: @account_name, storage_access_key: @account_key)
+  def self.connection
+    if File.exist?('../../admin/.env')
```

Comment:
Just so you know, @activemq, I am in the process of consolidating all `.env` files to a single one in the project root.  This is on #1478

Reply:
@jon-shipley understood, will make the necessary changes when 1478 is merged